src/bindings/capi/tvgCapi.cpp

```diff
@@ -255,6 +255,17 @@ TVG_EXPORT Tvg_Result tvg_shape_append_path(Tvg_Paint* paint, const Tvg_Path_Com
 }
 
 
+TVG_EXPORT Tvg_Result tvg_shape_get_path(const Tvg_Paint* paint, const Tvg_Path_Command** cmds, uint32_t* cmdCnt, const Tvg_Point** pts, uint32_t* ptsCnt)
```

Comment:
In my opinion it will be better to split tvg_shape_get_path to tvg_shape_get_path_commands and tvg_shape_get_path_cords. In some cases, for example if we want to check if two shapes are able to interpolate we need only path commands. In geometry_get api we need only path coordinates, so if we split tvg_shape_get_path as I mentioned earlier we can save some memory, but you should wait with fix for Mr. Hermet Park opinion.

Reply:
Both are fine to me.
In this case please check nullptr for each arguments and return value only if they are available.